javascript/javascript2/week1/danish.js

```diff
@@ -0,0 +1,27 @@
+function findNumberOfDanishLetters(str) {
+  let letterFrequency = {};
+  let totalOfLetters = 0;
+  const danishLetters = ["æ", "ø", "å"];
+  str = str.toLowerCase();
+  if (typeof str !== "string") {
+    return "Please input a string";
+  }
+  const words = str.split(" ");
+  words.forEach((str) => {
```

Comment:
```suggestion
  words.forEach((word) => {
```
To avoid confusing `str` with the other `str` above

Reply:
Also, this loop over each word is not necessary. You can get the same result with only `for (let char of str)`.